fe/shared/layouts/global.scss

```diff
@@ -143,24 +132,52 @@
     border-bottom-left-radius: 0;
     border-bottom-right-radius: 0;
 }
-    
-.react-autosuggest__suggestions-container {
-    display: none;
-}
 
-.react-autosuggest__suggestions-container--open {
-    display: block;
+.react-autosuggest__suggestions-container, .react-autosuggest__suggestions-container--open {
     position: absolute;
-    top: 2.5rem;
-    width: 100%;
+    top: 2rem;
+    left: calc(-19px + -1.6rem);
```

Comment:
**suggestion:** these calculations mixing rem and px units and minus values worry me, please stick to either px or rem